csrc/moe/moe_wna16.cu

```diff
@@ -304,8 +315,9 @@ torch::Tensor moe_wna16_gemm(torch::Tensor input, torch::Tensor output,
               "BLOCK_SIZE_K must divisible by group_size");
   TORCH_CHECK(BLOCK_SIZE_M <= 64, "BLOCK_SIZE_M must less or equal to 64");
   TORCH_CHECK(groups_per_block_row == 1 || groups_per_block_row == 2 ||
-                  groups_per_block_row == 4 || groups_per_block_row == 8,
-              "BLOCK_SIZE_K // group_size must be one of [1, 2, 4, 8]");
+                  groups_per_block_row == 4 || groups_per_block_row == 8 ||
+                  groups_per_block_row == 16,
+              "BLOCK_SIZE_K // group_size must be one of [1, 2, 4, 8, 16]");
```

Comment:
@nlzy forgot to mention: all the changes to `csrc/moe/moe_wna16.cu` are not in upstream. I had to make these changes specifically to get GLM 4.5 Air quants to work.